docs/content/getting-started/perform-list-users.mdx

```diff
@@ -294,27 +294,69 @@ With the tuples:
 
 Calling the List Users API for `document:1` with relation `viewer` of type `user` will yield the response below. It indicates that any object of type `user` (including those not already in OpenFGA as parts of tuples) has access to the system, except for a `user` with id `anne`.
 
-<ListUsersRequestViewer
-  authorizationModelId="01HXHKQX73VA6MJ3SRSY0D05VW"
-  objectType="document"
-  objectId="1"
-  relation="viewer"
-  userFilterType="user"
-  expectedResults={{
-    users: [{ wildcard: { type: "user" } }],
-    excluded_users: [{ object: { type: "user", id: "anne" } }]
-  }}
-  skipSetup={true}
-  allowedLanguages={[
-    SupportedLanguage.JS_SDK,
-    SupportedLanguage.GO_SDK,
-    SupportedLanguage.DOTNET_SDK,
-    SupportedLanguage.JAVA_SDK,
-    SupportedLanguage.CLI,
-    SupportedLanguage.CURL,
-  ]}
-/>
```

Comment:
The point of this block is to highlight the response only. The existing `<ListUsersRequestViewer/>` doesn't do a great job here so converted it to JSON.